App.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import List from './List'
+import Dashboard from './Dashboard'
+
+class App extends Component {
+  constructor() {
+    super;
```

Comment:
This should be invoked with props as well.
```suggestion
    super(props);
```